src/screens/LoginScreen.js

```diff
@@ -55,105 +51,99 @@ class LoginScreen extends Component {
         }
     };
 
-    render() {
-        const {containerStyle, textStyle, innerContainerstyle, buttonStyle, facebookStyle, loginContainer } = styles;
-        const {email, password} = this.props;
-        const{ hidden } = this.state
+    renderPage() {
+        const {appReady} = this.props;
+
+        if (!appReady) {
+            return <AppLoading />
+        }
```

Comment:
If `appReady === false` then show the Expo `AppLoading` component, which throws up the splash screen.